GUI/MainWindow.xaml

```diff
@@ -379,7 +380,7 @@
                     </Grid.RowDefinitions>
```

Comment:
Can you increase the Height to 60 on line 377? The additional line makes the text look cramped